packages/core/src/metas/registerInitial.ts

```diff
@@ -8,6 +8,9 @@ export function registerInitialMetas(faro: Faro): void {
     sdk: {
       version: VERSION,
     },
+    app: {
```

Comment:
Do you plan to provide the function to the outside or is it only for internal use?

If not, I would propose to move `faro.api.getBuildId(faro.config.app.name)` to a dedicated module, make it a utility function and remove it from the metas API.